python/ray/rllib/tuned_examples/ddpg_vs_ddpg2_tests/pendulum-ddpg2.yaml

```diff
@@ -0,0 +1,36 @@
+pendulum-ddpg2:
```

Comment:
Let's drop the parent directory from all these files?

Reply:
Yeah I can do that. The only thing is there already exists "montaincarcontinuous-ddpg.yaml" and "pendulum.ddpg" in /tuned_examples. Should I just replace those? 

The purpose of the yaml files in /ddpg_vs_ddpg2_tests was to compare the 2 implementations by using the (almost) same hyperparameters, not necessarily to find the configuration that performs best, which is why I decided to separate it from /tuned_examples. Also, should I remove the yaml files related to ddpg2? I left them just for comparison sake. Please let me know how you think we should proceed.

Reply:
Let's take out all the DDPG2 files. What are the performance differences for the current mountaincarcontinuous and pendulum yaml files compared to the new ones?